tests/acceptance/ZzzReinstallFromScratchCest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Models\Setting;
+use MailPoet\Test\DataFactories\Form;
+use MailPoet\Test\DataFactories\Newsletter;
+use MailPoet\Test\DataFactories\Segment;
+
+require_once __DIR__ . '/../DataFactories/Form.php';
+require_once __DIR__ . '/../DataFactories/Newsletter.php';
+require_once __DIR__ . '/../DataFactories/Segment.php';
+
+// This test should run last, because it removes all test data
```

Comment:
Other tests should be setting up their own data as they go, we really want to get away from building in implicit test order and dependencies, especially if we may want to introduce test running in parallel or randomize test order.

It is not necessary to name this test in such a way to get it to run at the end of the suite.